pkgs/applications/virtualization/colima/default.nix

```diff
@@ -1,14 +1,18 @@
 { lib
+, stdenv
 , buildGoModule
 , fetchFromGitHub
 , installShellFiles
 , lima
+, lima-bin
 , makeWrapper
 , qemu
 , testers
 , colima
 }:
 
+let lima-drv = if stdenv.isDarwin then lima-bin else lima;
```

Comment:
We should add a really short comment why we want this.

Reply:
@SuperSandro2000 Good idea. I added a comment and a link to the pull request for more infos.